src/libdecaf/src/gpu/opengl/opengl_driver.cpp

```diff
@@ -441,8 +441,7 @@ GLDriver::eventWrite(const pm4::EventWrite &data)
       decaf_abort(fmt::format("Unexpected event type {}", type));
    }
 
-   switch (data.addrLo.ENDIAN_SWAP())
-   {
+   switch (mPendingEOP.addrLo.ENDIAN_SWAP()) {
```

Comment:
Because I accidentally deleted too much code then undeleted it from the wrong place. Derp.